crates/bevy_transform/src/components/transform.rs

```diff
@@ -288,6 +288,37 @@ impl Transform {
         self.local_z()
     }
 
+    /// Moves the entity within its local space using the provided translation vector.
```

Comment:
Nit, I feel like these docs could be simplified. For example, the "Parameters" section seems unnecessary, and they could reference `Transform` (like the other docs) in place of "the entity".

Reply:
So I should change all "the entity" to `Transform`?